src/main/kotlin/com/avail/compiler/ParserState.kt

```diff
@@ -204,8 +203,8 @@ class ParserState internal constructor(
 	 *   A list of arbitrary [Avail values][AvailObject] that should be
 	 *   stringified.
 	 * @param transformer
-	 *   A [transformer][Transformer1] that accepts the stringified values and
-	 *   answers an expectation message.
+	 *   A transformer that accepts the stringified values and answers an
+	 *   expectation message.
```

Comment:
Down a few lines (GitHub won't let me tag them because they're unchanged), `transformer` is the last argument, but it's lined up with no visual separator with `expected`, which makes it hard to tell where the arguments end and the body begins.  I've been advocating hitting return between `String` and `) =` for cases like this.

Reply:
I reformatted to:

```kt
	internal fun expected(
			level: ParseNotificationLevel,
			values: List<A_BasicObject>,
			transformer: (List<String>)->String) =
		expected(level) { continuation ->
			Interpreter.stringifyThen(
				currentRuntime(),
				lexingState.compilationContext.textInterface,
				values
			) {
				continuation(transformer(it))
			}
		}
```

Objections?

Reply:
If Kotlin won't preserve the double-indented arguments (try cutting and pasting back the whole method, say), then I prefer the uglier form where ") =" goes to a new line.

Reply:
Ah, you're right. Okay, now it's:

```kt
	internal fun expected(
		level: ParseNotificationLevel,
		values: List<A_BasicObject>,
		transformer: (List<String>)->String
	) =
		expected(level) { continuation ->
			Interpreter.stringifyThen(
				currentRuntime(),
				lexingState.compilationContext.textInterface,
				values
			) {
				continuation(transformer(it))
			}
		}
```

That round-trips across cut and paste.